docs/modules/ROOT/pages/tutorial-kubernetes.adoc

```diff
@@ -144,8 +243,50 @@ spec:
             volumes:
                 - name: prometheus-config-volume
                   configMap:
-                      name: prometheus-config
----
+                      name: prometheus-config-cmos <.>
+----
+
+<.> `spec.template.spec.containers[0].image`: This is the CMOS image that we wish to run.
+<.> Refer to the logging section for more information
+<.> Refer to the disabling services section for more information
+
+Execute the below command to deploy cmos in the cluster
+[source,console]
+----
+$ kubectl apply -f cmos-workload.yaml
+----
+
+==== Disabling Service
+By default all the service of CMOS stack are enabled. You can choose to disable a service from being run, this is achieved by setting environment variables of the container. The naming convention to be followed to disable a service is `DISABLE_X`, where X is the name of the service.
```

Comment:
Please make sure to use single line per sentence to simplify reviews. It probably needs adding to the contribution guide so apologies for that.